go/ops.go

```diff
@@ -123,6 +129,16 @@ func (op *LeafOp) CheckAgainstSpec(spec *ProofSpec) error {
 
 // CheckAgainstSpec will verify the InnerOp is in the format defined in spec
 func (op *InnerOp) CheckAgainstSpec(spec *ProofSpec, b int) error {
+	if spec == nil || op == nil {
```

Comment:
Same comment here about check `op == nil`.

Reply:
The fuzzer flagged it.

Reply:
Done, thanks!